keel-api/src/main/kotlin/com/netflix/spinnaker/keel/rest/ResourceController.kt

```diff
@@ -102,4 +104,21 @@ class ResourceController(
     log.error(e.message)
     return mapOf("message" to (e.cause?.message ?: e.message))
   }
+
+  @ExceptionHandler(FailedValidationException::class)
+  @ResponseStatus(BAD_REQUEST)
```

Comment:
If this exception type means "I could parse it but it was logically wrong" (e.g. a cluster with a region that doesn't exist) this should be `UNPROCESSABLE_ENTITY`.